### PR TITLE
Style tables

### PR DESCRIPTION
**Pull request by dr15**
Styles table:
- Styles `WMTable`
- Adds cell components
- Updates mock data to use cell components

### CODE REVIEW

File: src/components/common/tableCells/DashCell.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { IDashCell } from './tableCells.interface';
+
+export default function DashCell({ value, children, className, ...otherProps }: IDashCell) {
+  const hasValue = value !== undefined && value !== null;
+
+  return hasValue ? (
```
Comment (nunibaranes) [inline src/components/common/tableCells/DashCell.tsx:8]: You can can set the value property in IDashCell as optional and change the type to ReactNode, and drop the variable hasValue, just check if the value exists. WDT?

File: src/components/common/tableCells/tableCells.interface.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  value: string;
+  color: string;
+}
+
+export interface ITextArrayCell extends ITableCell {
+  value: Array<string>;
+}
+
+export interface ITextCell extends ITableCell {
+  value: string | React.ReactNode;
```
Comment (nunibaranes) [inline src/components/common/tableCells/tableCells.interface.ts:42]: ReactNode includes string type.

Comment (dr15) [reply src/components/common/tableCells/tableCells.interface.ts]: Fixed

File: src/components/common/tableCells/DashCell.tsx (hunk 1 above)
Comment (dr15) [reply src/components/common/tableCells/DashCell.tsx:8]: I improved it a bit

**Pull request merged**